openapi.yaml

```diff
@@ -12,106 +12,206 @@ info:
-        - in: query
-          name: filter[age][lt]
-          description: Filter results by age
-          required: false
-          schema:
-            type: number
-            format: integer
       responses:
         '200':
           description: Successful response
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/PetSetResult'
+                $ref: '#/components/schemas/PositionResult'
+        '404':
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+  /hr/departments:
+    get:
+      summary: List departments
+      tags:
+        - departments
+      description: Return departments for a single business center
+      operationId: getDepartments
+      parameters:
+        - $ref: '#/components/parameters/businessCenter'
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DepartmentSetResult'
         '400':
           $ref: '#/components/responses/400'
         '500':
           $ref: '#/components/responses/500'
-    post:
-      summary: Add a new pet
+  /hr/departments/{departmentId}:
+    get:
+      summary: Get department by ID
       tags:
-        - pets
-      operationId: postPet
-      requestBody:
-        required: true
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/PetPostBody'
+        - departments
+      description: Return a single department.
+      operationId: getDepartmentById
+      parameters:
+        - name: departmentId
+          in: path
+          description: Unique department ID
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DepartmentResult'
+        '404':
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+  /hr/locations:
+    get:
+      summary: List locations
+      tags:
+        - locations
+      description: >-
+        Return a list of HR locations that can be related to a job record, including minimum wages
+        if applicable.
+      operationId: getLocations
+      parameters:
+        - name: filter[minimumWageDate]
+          in: query
+          description: Date to search by for minimum wage data. Minimum wage depends on the date.
+          required: false
+          schema:
+            type: string
+            format: date
+            example: '2018-12-31'
+        - name: filter[name]
+          in: query
+          description: Search by location name
+          required: false
+          schema:
+            type: string
+            example: Corvallis
```

Comment:
Some of these need examples. I think this one should go on `city` and a different one put on `name`

Reply:
Well, I think the name for Corvallis location is also Corvallis. This is what I got from v1:

```json
{
            "id": "02A",
            "type": "Location",
            "attributes": {
                "name": "Corvallis",
                "city": "Corvallis",
                "county": "Benton County",
                "state": "Oregon",
                "stateCode": "OR",
                "minimumWageClassification": "Standard",
                "minimumWage": 11.25
            },
            "links": {
                "self": "https://oregonstateuniversity-test.apigee.net/v1/hr/locations/02A?date=2020-05-19"
            }
        },
```